### PR TITLE
C# client

### PR DESCRIPTION
**Pull request by ahimta**

### CODE REVIEW

File: Account.cs [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+	internal class DepositAdressesRequest : PrivateRequest { }
+
+	[DataContract]
+	public class DepositAddressesResponse : MexbtResponse
+	{
+		[DataMember(Name = "addresses")] public Address[] Addresses { get; set; }
+	}
+
+	[DataContract]
+	internal class DepositAdressByCurrencyRequest : PrivateRequest { }
```
Comment (williamcoates) [inline Account.cs]: typo: Adress -> Address

File: Account.cs [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+
+	public class DepositAdressByCurrencyResponse : MexbtResponse
```
Comment (williamcoates) [inline Account.cs]: Address*

File: Account.cs [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+
+	[DataContract]
+	public class Address
+	{
+		[DataMember(Name = "depositAddress")] public string DepositAddress { get; set; }
+		[DataMember(Name = "name")]           public string Name           { get; set; }
+	}
+
+	[DataContract]
+	internal class DepositAdressesRequest : PrivateRequest { }
```
Comment (williamcoates) [inline Account.cs]: Addresses*

**Comment by ahimta** [comment]
Fixed the typos.

**Pull request merged**